CMSgov/mint-app
# [EASI-4076] Updating dependencies

# EASI-4076


## Changes and Description

- Updated Luxon to latest
- Updated Vite to 4.5.2 (v5.x.x requires Node 18)
- Updated Axios to 1.6.7



## How to test this change

1. `yarn install`
2. `scripts/dev down && scripts/dev up` 
3. Verify the entire app is running fine
4. try to break app



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
